docs/modules/general/pages/local-env.adoc

```diff
@@ -0,0 +1,27 @@
+= OneCX Local Environment
+:local-env-url: https://github.com/onecx/onecx-local-env?tab=readme-ov-file#onecx-local-environment-setup
+
+== Overview
+With the OneCX platform, every Application needs to adjusted so it can be integrated within the platform. For the best experience with Application development and integration, OneCX offers a {local-env-url}[OneCX Local Enviornment] repository. It provides a consistent setup of the platform and Core Applications with minimal actions and allows extension.
+
+== Setting Up OneCX and Core Applications
+Using {local-env-url}[OneCX Local Enviornment] repository allows to:
+
+* set up all required services to run the OneCX platform
+* set up Shell and other Core Applications with preconfigured data
+* develop and test Applications already integrated with the platform
```

Comment:
suggestion: develop and test Applications which are already integrated with the platform